doc/reference/data.rst

```diff
@@ -37,7 +37,29 @@ Functions to build a :py:class:`DataPipeline`:
     Collater
     CollateOptionsOverride
 
+Column syntax
+~~~~~~~~~~~~~
+
+The data items going through the pipeline don't have to be flat tensors, but can be tuples, or python dictionaries.
+Several operators have a syntax to specify a specific column of the input data.
+Notably the :py:func:`DataPipelineBuilder.map` operator
+has a `selector` argument to choose the column to apply the function to.
+
+If the data item is a tuple,
```

Comment:
Nit: Maybe this is something worth to include in the contributor guidelines; ideally comments and lines in text files should wrap by 80th char (i.e. line length = 80). I see that the lines in this file have pretty inconsistent lengths. It would be nice to align them around 80.

Reply:
My strategy is to not worry about indentation here, since I expect people will mostly read it in HTML format.
If you have a formatter that handle that, why not, but otherwise it's too tedious to keep it manually aligned.

I did try to improve a few over-split sentences.